main.js

```diff
@@ -0,0 +1,248 @@
+let start = document.getElementById("start"); //to start game and timer?
+// let stop = document.getElementById("stop"); //this is to stop clock...
```

Comment:
In your declare winner function that highlights the winner on the board, you could add something within that like `.clearInterval()`